core/shared/src/main/scala/scalaz/zio/Task.scala

```diff
@@ -0,0 +1,341 @@
+package scalaz.zio
+
+import scalaz.zio.Exit.Cause
+import scalaz.zio.internal.{ Executor, Platform }
+
+import scala.concurrent.ExecutionContext
+
+object Task {
```

Comment:
I would alphabetize all the methods in this calss for easy lookup / better organization.